Mini-Rewrite of Skinning Tutorial (#2739)
ppy/osu-wiki: wiki/Skinning_Tutorial/en.md

*For frequently answered skinning questions, see: [Skinning/FAQ](/wiki/Skinning/FAQ)*

## Getting started

> To get started, open File Explorer. enter the path where you installed osu!, and press enter. Then, go into the `osu!/` folder, then `Skins/`. Right click, and create a new folder. You can also go into osu!, settings, and open the skins folder, then make the folder. You can name it whatever you want, but keep in mind that osu! will display this name on the dropdown of the skin select. Inside this folder, you can add any of the skinning elements listed in the [Skinning Elements Lists](/wiki/Skinning#skin-elements-lists).

Reviewer: Revised version is too complex, revert this.

> Contrary to popular belief, to get a better understanding of how skinning works, you should start by mixing skins (copying other skin assets and pasting them into your skin folder). **You are not allowed to share these in the forums.** However, mixing skins for yourself can help you learn what skinning can and can not achieve in osu!.

Reviewer: Bolded reminder is too imposing. Recommended you rewrite this a reminder instead.

> ## Getting Started

Reviewer 2: ```suggestion
## Getting started
```

> To get started, open File Explorer. enter %localappdata% on the path bar, and press enter. Then, go into the `osu!/` folder, then `Skins/`. Right click, and create a new folder. You can also go into osu!, settings, and open the skins folder, then make the folder. You can name it whatever you want, but keep in mind that osu! will display this name on the dropdown of the skin select. Inside this folder, you can add any of the skinning elements listed in the [Skinning Elements Lists](/wiki/Skinning#skin-elements-lists).

Reviewer 2: shouldn't assume where the game is installed, the gist of the old version is fine for finding the skins folder (language is oddly casual, though)

> Contrary to popular belief, to get a better understanding of how skinning works, you should start by mixing skins (copying other skin assets and pasting them into your skin folder). **You are `not` allowed to share these in the forums.** However, mixing skins for yourself can help you learn what skinning can and can not achieve in osu!.

Reviewer 2: ```suggestion
Contrary to popular belief, to get a better understanding of how skinning works, you should start by mixing skins (copying other skin assets and pasting them into your skin folder). **You are not allowed to share these in the forums.** However, mixing skins for yourself can help you learn what skinning can and can not achieve in osu!.
```
code blocks aren't for emphasis (and in this case I don't think any is needed)

> ## Making A Skin

Reviewer 2: ```suggestion
## Making a skin
```
headings should be sentence cased

---

> The most important thing you will need is a theme or an idea. You should have one in mind to create an objective for what you want your skin to look like. This is also to help yourself when you need to gather and/or create assets.

Reviewer: Theme is rather not so important as skins are mostly aimed for tailoring the user experience for the author.

> The last thing to have ready is patience and lots of time. New skins do not complete in a snap but can take a week or even several months, so give it some time.

Reviewer 2: not sure what the point is

## What's next?

> From here, you can checkout the [Skinning Elements Lists](/wiki/skinning#skin-elements-lists) to get an idea for what to include in your skin or go through the some of the tutorials:

Reviewer 2: ```suggestion
From here, you can check out the [skin elements lists](/wiki/Skinning#skin-elements-lists) to get an idea for what to include in your skin, or go through the some of the tutorials:
```
